File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { PassantenfrequenzStadtStgallenSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await PassantenfrequenzStadtStgallenSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- PassantenfrequenzStadtStgallen SDK exists test

local sdk = require("passantenfrequenz-stadt-stgallen_sdk")

describe("PassantenfrequenzStadtStgallenSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
